src/dropdown-arrow-down.js

```diff
@@ -0,0 +1,13 @@
+import { createElement, Component } from 'preact' /** @jsx createElement */
+
+export default class DropdownArrowDown extends Component {
```

Comment:
I think I'd rather keep dropdown as that's the specific context it's designed for - a more generic down icon (say for sorting) might be different